source/tutorials/first-steps/declarative-shell.md

```diff
@@ -26,9 +26,9 @@ In this tutorial we'll take a look at how to create reproducible shell environme
 
 30 minutes
 
-### What will you need?
+### What do you need?
 
-- A basic understanding of the [Nix language](reading-nix-language)
+- A very basic understanding of the [Nix language](reading-nix-language)
```

Comment:
```suggestion
- A basic understanding of the [Nix language](reading-nix-language)
```

Reply:
I wanted to stress that it's much less than in the tutorial. Maybe "rudimentary"?

Reply:
```suggestion
- A rudimentary understanding of the [Nix language](reading-nix-language)
```